Surface_mesh_topology/test/Surface_mesh_topology/path_tests.cpp

```diff
@@ -93,7 +93,7 @@ bool basic_tests()
 
   internal::Light_MQ<CMap> lmq(p6.get_map());
   internal::Path_on_surface_with_rle<internal::Light_MQ<CMap> > p7(lmq, p6);
-  if (!p7.is_valid() || p7.size_of_list()!=2)
+  if (!p7.is_valid() || p7.size_of_list()!=3)
```

Comment:
the cerr has not been updated

Reply:
Addressed in https://github.com/CGAL/cgal/pull/4898/commits/37d55a466f8911484dad7d5f705b74b9360121f5